logstash-input-kinesis.gemspec

```diff
@@ -20,9 +20,11 @@ Gem::Specification.new do |spec|
 
   spec.platform      = 'java'
 
-  spec.requirements << "jar 'com.amazonaws:amazon-kinesis-client', '1.13.3'"
-  spec.requirements << "jar 'com.amazonaws:aws-java-sdk-core', '1.11.1034'"
-  spec.requirements << "jar 'com.amazonaws:aws-java-sdk-sts', '1.11.1034'"
+  spec.add_runtime_dependency 'logstash-core', '>= 8.9.0'
```

Comment:
Can you explain why the plugin requires core >= 8.9.0?

Reply:
Because amazon-kinesis-client 1.13.3 transitively depends on jackson-databind 2.15.2 which is bundled in Logstash. The same version of the library is shipped starting from Logstash 8.9.0. Despite the plugin could also work with other versions, we don't have the ability to grant the order in which the libraries are loaded, so it could end that in another Logstash it loads upfront the library shipped with this plugin instead of the one it's bundled with, and could cause binary incompatibilities at runtime.